src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -187,7 +187,22 @@ export default {
 		loanChannelQueryMapMixin
 	],
 	watch: {
-		selectedCategory() {
+		selectedCategory(categoryId) {
+			const catId = Number(categoryId);
+			const queryMap = loanChannelQueryMapMixin.data().loanChannelQueryMap;
+			const categoryFilter = catId === 0 ? {} : queryMap
+				.find(channel => channel.id === catId)?.flssLoanSearch;
+			// These categories use location/gender/sort by for FLSS and need
+			// to have dropdowns preset
+			if (catId === 5) { // women
+				this.selectedGender = 'female';
+			} else if (catId === 28) { // kiva-u-s
```

Comment:
Are there other special cases coming? If so, there probably needs to be a programmatic approach to setting the other filters, rather than special cases. Otherwise if there are just these 3 for the time being, fine to leave as is.

Reply:
Should just be these 3 for now, unless we add new categories to Kiva with similar presets!